app/helpers/application_helper.rb

```diff
@@ -1,5 +1,17 @@
 # frozen_string_literal: true
 module ApplicationHelper
+  NAMES = { "local" => "LOC:",
+                "dev" => "DEV:",
+                "staging" => "STG:",
+                "test" => "TST:",
+                "production" => ""
+  }
+  def env_title_prefix
+    env_var = Figaro.env.HTML_TITLE_PREFIX_ENV
```

Comment:
`Figaro.env` is just a proxy for ENV, and it's preference, so I'm using it b/c it looks nicer.